packages/docusaurus-theme-translations/locales/uz/theme-search-algolia.json

```diff
@@ -0,0 +1,35 @@
+{
+  "theme.SearchBar.label": "Qidiruv",
+  "theme.SearchBar.seeAll": "Barcha natijalarni ko‘rish ({count})",
+  "theme.SearchModal.errorScreen.helpText": "Internet aloqangizni tekshiring.",
+  "theme.SearchModal.errorScreen.titleText": "Qidiruv natijalarini yuklab bo‘lmadi",
+  "theme.SearchModal.footer.closeKeyAriaLabel": "Escape tugmasi",
+  "theme.SearchModal.footer.closeText": "yopish",
+  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Pastga strelka tugmasi",
+  "theme.SearchModal.footer.navigateText": "navigatsiya",
+  "theme.SearchModal.footer.navigateUpKeyAriaLabel": "Yuqoriga strelka tugmasi",
+  "theme.SearchModal.footer.searchByText": "dan qidiruv",
+  "theme.SearchModal.footer.selectKeyAriaLabel": "Enter tugmasi",
+  "theme.SearchModal.footer.selectText": "tanlash",
+  "theme.SearchModal.noResultsScreen.noResultsText": "Qidiruv natijalari yo‘q",
+  "theme.SearchModal.noResultsScreen.reportMissingResultsLinkText": "Bizga xabar bering.",
+  "theme.SearchModal.noResultsScreen.reportMissingResultsText": "Mos keladigan qidiruv natijasi topilmadimi?",
+  "theme.SearchModal.noResultsScreen.suggestedQueryText": "Harakat qib ko'ring",
+  "theme.SearchModal.placeholder": "QIdiruv",
+  "theme.SearchModal.searchBox.cancelButtonText": "Rad etish",
+  "theme.SearchModal.searchBox.resetButtonTitle": "Tozalash",
+  "theme.SearchModal.startScreen.favoriteSearchesTitle": "Sevimlilar",
+  "theme.SearchModal.startScreen.noRecentSearchesText": "Qidiruv tarixi yo'q",
+  "theme.SearchModal.startScreen.recentSearchesTitle": "Yaqinda",
+  "theme.SearchModal.startScreen.removeFavoriteSearchButtonTitle": "Sevimlilardan olib tashlash",
+  "theme.SearchModal.startScreen.removeRecentSearchButtonTitle": "Tarixdan yozuvni o'chirish",
+  "theme.SearchModal.startScreen.saveRecentSearchButtonTitle": "Qidiruvingizni saqlang",
+  "theme.SearchPage.algoliaLabel": "Algolia dan qidiruv",
+  "theme.SearchPage.documentsFound.plurals": "{hisob} hujjat|{hisoblash} hujjat|{hisoblash} hujjat",

```

Comment:
The placeholder should remain `{count}`

Also, don't we say "3 ta hujjat topildi" instead of just "3 hujjat"?

Reply:
You're right. Should be {count} hujjat|{count} ta hujjat|{count} ta hujjat